app/views/govuk_publishing_components/components/_inverse_header.html.erb

```diff
@@ -1,10 +1,12 @@
 <%
   full_width ||= false
   header_class = full_width ? "gem-c-inverse-header--full-width" : ""
+  no_padding_top ||= false
+  padding_top_class = no_padding_top ? "gem-c-inverse-header--no-padding-top" : ""
```

Comment:
This reads a little strangely to me. If you pass `padding_top` == true, then you render the component without padding. Is that right?